artemis-distribution/src/main/resources/licenses/bin/LICENSE

```diff
@@ -432,4 +432,46 @@ For Underscore:
 ==============================================================================
 This product bundles Underscore, which is available under a
 "MIT" license.  For details, see
-https://github.com/jashkenas/underscore/blob/master/LICENSE
\ No newline at end of file
+https://github.com/jashkenas/underscore/blob/master/LICENSE
+
+==============================================================================
```

Comment:
Looking at the existing file I would suggest adding all these additions above the previous 'grouping' of console bits to be clearer that they are separate, up at https://github.com/apache/activemq-artemis/blob/2.17.0/artemis-distribution/src/main/resources/licenses/bin/LICENSE#L247

Reply:
I'm not sure we should move above `Apache ActiveMQ Artemis Subcomponents` https://github.com/apache/activemq-artemis/blob/2.17.0/artemis-distribution/src/main/resources/licenses/bin/LICENSE#L247
maybe above `For Angular` https://github.com/apache/activemq-artemis/blob/2.17.0/artemis-distribution/src/main/resources/licenses/bin/LICENSE#L264
WDYT?

Reply:
I dont think it should go there as that would still be suggesting they are part of the 'management console component' demarcated by https://github.com/apache/activemq-artemis/blob/2.17.0/artemis-distribution/src/main/resources/licenses/bin/LICENSE#L256 when they aren't. I think above 247 makes sense.

Reply:
fixed